acracensor/acracensor_interfaces.go

```diff
@@ -3,7 +3,7 @@ package acracensor
 type QueryHandlerInterface interface {
 	CheckQuery(sqlQuery string) error
 	Reset()
-	GetName() string
+	//GetName() string
```

Comment:
remove?

Reply:
Yeah. It is redundant method - censor doesn't use it